tests/settings_tests/tests.py

```diff
@@ -443,9 +443,9 @@ def test_module(self):
             del sys.modules['fake_settings_module']
 
     def test_override(self):
-        self.assertFalse(settings.is_overridden('ALLOWED_HOSTS'))
-        with override_settings(ALLOWED_HOSTS=[]):
-            self.assertTrue(settings.is_overridden('ALLOWED_HOSTS'))
+        self.assertFalse(settings.is_overridden('PREPEND_WWW'))
+        with override_settings(PREPEND_WWW=True):
+            self.assertTrue(settings.is_overridden('PREPEND_WWW'))
```

Comment:
FYI, the only reason for this change is that ALLOWED_HOSTS is now modified by default in ``test_sqlite.py``